config/am.py

```diff
@@ -0,0 +1,58 @@
+"Amharic config with language-specific information."
+
+from pynini import *
+
+GRAPHEMES = union("'", "-", "ʼ",
```

Comment:
Just checking that this is indeed supposed to be MODIFIER_LETTER_APOSTROPHE :)

Reply:
Oops! This _is_ supposed to be there in Hausa, but I inadvertently added it to Amharic and Igbo as well.